### PR TITLE
Polish looper init tutorial and looper init_piface

### PR DESCRIPTION
**Pull request by donaldcampbelljr**
Works towards addressing: #466 

`looper init_piface` simply generates a generic piface.

`looper init` now gives the options of automated/generic looper config vs following a step by step guide where the user can copy and paste some elements.

Used rich to add a bit of color and rulers to make the process a bit more visually pleasing and polished,e.g.
image
image

### CODE REVIEW

File: looper/cli_pydantic.py [hunk 1]
```diff
@@ -122,17 +124,34 @@ def run_looper(args: TopLevelParser, parser: ArgumentParser, test_args=None):
...
+        console.clear()
+        console.rule(f"\n[magenta]Looper initialization[/magenta]")
+        console.print(
+            "[bold]Would you like to follow a guided tutorial?[/bold]  [green]Y[/green] / [red]n[/red]..."
         )
 
+        selection = None
+        while selection not in ["y", "Y", "n", "N"]:
+            selection = console.input("\nSelection: ")
+
```
Comment (khoroshevskyi) [inline looper/cli_pydantic.py:138]:
```suggestion
        selection = None
        while selection not in ["y", "n"]:
            selection = console.input("\nSelection: ").lower().strip()

```

File: looper/cli_pydantic.py [hunk 2]
```diff
@@ -122,17 +124,34 @@ def run_looper(args: TopLevelParser, parser: ArgumentParser, test_args=None):
...
+        if selection in ["n", "N"]:
```
Comment (khoroshevskyi) [inline looper/cli_pydantic.py:139]:
```suggestion
        if selection == "n":
```

File: looper/utils.py [hunk 3]
```diff
@@ -521,18 +549,146 @@ def initiate_looper_config(
...
+    console.print(
+        f"Initialized looper config file: [yellow]{looper_config_path}[/yellow]"
+    )
+
+    return True
+
+
+def looper_config_tutorial():
+    # Prompt a user through configuring a .looper.yaml file for a new project.
+    # To be used in as an option for `looper init`.
```
Comment (khoroshevskyi) [inline looper/utils.py:580]: add docstring and if it returns anything

**Review by khoroshevskyi — APPROVED** [review]
LGTM

File: looper/utils.py (hunk 3 above)
Comment (donaldcampbelljr) [reply looper/utils.py]: Great, this is done.

File: looper/cli_pydantic.py (hunk 1 above)
Comment (donaldcampbelljr) [reply looper/cli_pydantic.py]: I fixed this is another place as well to make it cleaner.

**Pull request merged**